src/lab1/structures/vector/Vector3D.ts

```diff
@@ -11,6 +16,17 @@ export default class Vector3D {
     this.z = z;
   }
 
+  public rotate(angleX: number, angleY: number, angleZ: number): Vector3D {
+    return transformVector(
+      this,
+      transformations.rotate3d(angleX, angleY, angleZ)
+    );
+  }
+
+  public scale(x: number, y: number, z: number): Vector3D {
+    return transformVector(this, transformations.scale3d(x, y, z));
+  }
```

Comment:
Where is a translating? Translating here would mean translating of line segment (between start and end point of vector)